packages/graph/hash_graph/lib/graph/src/store/postgres/mod.rs

```diff
@@ -787,62 +787,65 @@ where
         //   see https://app.asana.com/0/1202805690238892/1203277018227719/f
         // TODO: `collect` is not needed but due to a higher-ranked lifetime error, this would fail
         //       otherwise. This is expected to be solved in future Rust versions.
-        let entity_type_references = entity_type
-            .link_mappings()
-            .into_keys()
-            .chain(
-                entity_type
-                    .link_mappings()
-                    .into_values()
-                    .flatten()
-                    .flatten(),
-            )
-            .chain(entity_type.inherits_from().all_of())
-            .collect::<Vec<_>>();
-
-        let entity_type_reference_ids = self
-            .entity_type_reference_ids(entity_type_references)
-            .await
-            .change_context(InsertionError)
-            .attach_printable("Could not find referenced entity types")?;
-
-        for target_id in entity_type_reference_ids {
-            self.as_client()
-                .query_one(
-                    r#"
-                        INSERT INTO entity_type_entity_type_references (
-                            source_entity_type_version_id,
-                            target_entity_type_version_id
-                        )
-                        VALUES ($1, $2)
-                        RETURNING source_entity_type_version_id;
-                    "#,
-                    &[&version_id, &target_id],
-                )
-                .await
-                .into_report()
-                .change_context(InsertionError)?;
-        }
+        // Currently we dont have a `entity_type_entity_type_references` table. These checks
```

Comment:
Do we maybe want to keep this on as `owned_entity_type_entity_type_references` or something to make it easier to transition in the mean time?

If we really can use JSON path queries for all the edge kinds (not just inheritance) then I guess this is fine, but I had assumed that'd be even more work.

The suggestion of `owned_entity_type_entity_type_references` maybe wouldn't work for long, perhaps it'd need to be `owned_entity_type_closed_entity_type_references` or something

Reply:
Reverted in b91427ac89be33251f6bc7158b6a79b2a0442e06